lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3463,6 +3476,10 @@ namespace {
         result->addMember(method);
       }
 
+      if (destructor) {
+        result->addMember(destructor);
```

Comment:
Not sure I understand what this is doing -- I thought only classes were allowed to have deinitializers in Swift?